atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -148,7 +148,6 @@
 import com.palantir.util.OptionalResolver;
 
 @Value.Immutable
-@Value.Style(stagedBuilder = true)
```

Comment:
fwiw, the staged builder bit was _great_; meant it was basically impossible to construct the txn manager wrong